<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>925</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_trem1">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: purple}</string>
    </property>
    <property name="text">
     <string>T1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>380</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ligar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>120</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>50</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: darkorange}</string>
    </property>
    <property name="text">
     <string>T2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="text">
     <string>T3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho4_3">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>120</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho2_3">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho1_5">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>250</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trilho3_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>21</width>
      <height>137</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: yellow}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: red}</string>
    </property>
    <property name="text">
     <string>T4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_trem5">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>180</y>
      <width>21</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: green}</string>
    </property>
    <property name="text">
     <string>T5</string>
    </property>
   </widget>
   <widget class="QSlider" name="T1Slider">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="valorSlider">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="valorSlider_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 2</string>
    </property>
   </widget>
   <widget class="QSlider" name="T2Slider">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="valorSlider_3">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>300</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 3</string>
    </property>
   </widget>
   <widget class="QSlider" name="T3Slider">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>320</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="valorSlider_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>300</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 4</string>
    </property>
   </widget>
   <widget class="QSlider" name="T4Slider">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>320</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="valorSlider_5">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>300</y>
      <width>58</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trem 5</string>
    </property>
   </widget>
   <widget class="QSlider" name="T5Slider">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>320</y>
      <width>160</width>
      <height>28</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { background: blue}</string>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>label_trilho3_2</zorder>
   <zorder>label_trilho1_2</zorder>
   <zorder>label_trilho1</zorder>
   <zorder>label_trilho4_2</zorder>
   <zorder>label_trilho2_3</zorder>
   <zorder>label_trilho4_3</zorder>
   <zorder>label_trilho1_3</zorder>
   <zorder>label_trilho3_4</zorder>
   <zorder>label_trilho1_5</zorder>
   <zorder>label_trilho3_3</zorder>
   <zorder>label_trilho1_4</zorder>
   <zorder>label_trilho2</zorder>
   <zorder>label_trilho3</zorder>
   <zorder>label_trilho4</zorder>
   <zorder>label_trem1</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>label_trilho2_2</zorder>
   <zorder>label_trem2</zorder>
   <zorder>label_trem3</zorder>
   <zorder>label_trem4</zorder>
   <zorder>label_trem5</zorder>
   <zorder>T1Slider</zorder>
   <zorder>valorSlider</zorder>
   <zorder>valorSlider_2</zorder>
   <zorder>T2Slider</zorder>
   <zorder>valorSlider_3</zorder>
   <zorder>T3Slider</zorder>
   <zorder>valorSlider_4</zorder>
   <zorder>T4Slider</zorder>
   <zorder>valorSlider_5</zorder>
   <zorder>T5Slider</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>925</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
